images/nginx/rootfs/build.sh

```diff
@@ -241,47 +226,36 @@ if [[ ${ARCH} == "x86_64" ]]; then
   luarocks install lrexlib-pcre 2.7.2-1
 fi
 
-# luajit is not available on ppc64le and s390x
-if [[ (${ARCH} != "ppc64le") && (${ARCH} != "s390x") ]]; then
-  cd "$BUILD_PATH/luajit2-2.1-20180420"
-  make CCDEBUG=-g
-  make install
```

Comment:
related to https://github.com/kubernetes/ingress-nginx/pull/3214/files#r223915810, why are you removing this?